manifests/f/FreeCiv/FreeCiv/3.0.7/FreeCiv.FreeCiv.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FreeCiv.FreeCiv
+PackageVersion: 3.0.7
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-04-07
+Installers:
+- Architecture: x64
+  InstallerUrl: https://sourceforge.net/projects/freeciv/files/Freeciv 3.0/3.0.7/Freeciv-3.0.7-msys2-win64-gtk3.22-setup.exe/download
+  InstallerSha256: EC09887CD9F895CD078C7B3889F3CCC943968007964218475501B3F907620649
+  ProductCode: Freeciv-3.0.1-gtk3

```

```suggestion
  ProductCode: Freeciv-3.0.7-gtk3
```
I haven't tested, but it's likely this is the ProductCode